text/0000-machine-readable-tests-output.md

```diff
@@ -0,0 +1,146 @@
+- Feature Name: machine\_readable\_tests\_output
+  - Start Date: 2015-09-17
+  - RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Replace current test output with machine-readable one and add thin compatibility
+layer on top of that.
```

Comment:
> and add thin compatibility layer on top of that.

What do you mean by "compatibility layer"? The JSON-based output (instead of binary Rust structs)? In _Drawbacks_ you write that new tooling will be required for compatibility but this sentence sounds to me like that tooling is actually part of the RFC.